torch/fx/experimental/proxy_tensor.py

```diff
@@ -588,9 +590,13 @@ def wrapped(*args):
         else:
             raise AssertionError(f"Unexpected tracing type: {tracing_mode}")
 
+        python_dispatch_table = {key: value for key, value in decomposition_table.items() if type(key) is tuple}
```

Comment:
It's a bit hard for me to tell, just from the change here, what the expected type of `decomposition_table` is now

Reply:
decomposition_table's key is either an OpOverload, or a tuple of (OpOverload, DispatchKey)...